app/middleware/Helper.js

```diff
@@ -0,0 +1,28 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const SECRET = process.env.SECRET;
+
+function generateHashPassword(password) {
+  return bcrypt.hashSync(password, bcrypt.genSaltSync(10));
+}
+
+function comparePassword(hashPassword, password) {
+  return bcrypt.compareSync(password, hashPassword);
+}
+
+function generateToken(id) {
```

Comment:
Could you convert all these function to arrow functions

Reply:
I have converted to arrow function